### PR TITLE
Fix GitVersionSettings.UpdateAssemblyInfoFileName to be list of file paths

### PR DESCRIPTION
**Pull request by gebolze**
I confirm that the pull-request:

- [x] Follows the contribution guidelines
- [x] Is based on my own work
- [x] Is in compliance with my employer

### CODE REVIEW

File: build/specifications/GitVersion.json [hunk 1]
```diff
@@ -76,9 +76,10 @@
           },
           {
             "name": "UpdateAssemblyInfoFileName",
```
Comment (matkoch) [inline build/specifications/GitVersion.json]: UpdateAssemblyInfoFileNameS, plural

**Comment by matkoch** [comment]
@gebolze thanks for checking on this! Except from the naming, everything's looking fine.

File: build/specifications/GitVersion.json (hunk 1 above)
Comment (gebolze) [reply build/specifications/GitVersion.json]: I've updated the PR according to your comment

**Comment by matkoch** [comment]
Thanks for your contribution! 😄🎉

**Pull request merged**